src/main/scala/cromwell/Main.scala

```diff
@@ -203,11 +256,35 @@ object Main extends App {
         |  Starts a web server on port 8000.  See the web server
         |  documentation for more details about the API endpoints.
       """.stripMargin)
-    if (exit) System.exit(-1)
+    exit(-1)
   }
 
-  def getAction(firstArg: Option[String]): Option[Actions.Value] = for {
+  private[this] def continueIf(valid: => Boolean)(block: => Int): Int = if (valid) block else usageAndExit()
```

Comment:
Could be made generic, though I'm not sure that would broaden its applicability here.